### DISCUSSION TITLE
Selective Backups

### DISCUSSION THREAD

**Discussion by addy419** [nextcloud/all-in-one#3819] [Questions] [answered]
Hi,

When looking at the logs, it looks like the borg backup backs up all my data as well? Is that the case?
If so, is it possible to set it so that only the database and some containers are backed up. I don't want the backup to have the same size as my data.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#what-is-getting-backed-up-by-aios-backup-solution
